app/src/test/java/org/simple/clinic/editpatient/PatientEditScreenControllerTest.kt

```diff
@@ -445,4 +476,220 @@ class PatientEditScreenControllerTest {
         listOf(PatientEditGenderChanged(Gender.TRANSGENDER), emptySet<PatientEditValidationError>())
     )
   }
+
+  @Test
+  @Parameters(method = "params for saving patient on save clicked")
+  fun `when save is clicked, the patient details must be updated if there are no errors`(
```

Comment:
This test and the corresponding parameters are quite extensive. I'd like some feedback on this if it can be simplified further.

Reply:
I'm wondering if I should split it into two tests, one where a number already exists and another where the number doesn't. I don't know if it's worth it though, since the only thing it saves is an `if` clause in the test and a lot of duplicated test parameters.